README.md

```diff
@@ -1,4 +1,4 @@
-# superchain-registry
+<img width="3200" alt="Frame 2 (5)" src="https://github.com/user-attachments/assets/088a3923-762f-41fb-bd0a-975dea417e7f"><br/>
```

Comment:
Can the file be put into the `.github` directory and referenced that way? The `github.com/user-attachments` asset link might break. Also updating the `alt` image text while here would be nice.

Reply:
That would indeed be nice to fix! Otherwise great banner ✨🔴⏡🔴✨

Reply:
Done! https://github.com/ethereum-optimism/superchain-registry/pull/474/commits/775076127721b6551eb31cfe268ec61382739710